class.wp-cldr.php

```diff
@@ -309,7 +315,7 @@ public function flush_all_wp_caches() {
 	 * @return array An associative array where keys are WordPress locales and values are CLDR data items
 	 */
 	private function get_locale_bucket( $locale, $bucket ) {
-		if ( null === $locale ) {
+		if ( '' === $locale ) {
```

Comment:
Is the `null` check no longer desirable here at all? Is there any reason we couldn't just check `empty( $locale )` instead?